docs/UserGuide.md

```diff
@@ -37,24 +33,118 @@ You can also generate a `CSV` payslip for your employees to refer to.
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: If this does not work, open an instance of the command line (command prompt for Windows, the terminal on Mac and Linux), navigate to directory where the file is located, and run `java -jar peoplesoft.jar`.
+
+</div>
 
-4. Double-click the file to start the app. The GUI should appear as pictured in a few seconds. If this does not work on your system, open an instance of the command line (command prompt for Windows, the terminal on Mac and Linux), navigate to directory where the file is located, and type `java -jar peoplesoft.jar`. Note how the app contains some sample data.
 
-<br><br>
 
 ![Ui](images/Ui.png)<br>
 _The PeopleSoft interface_
 
-<br><br>
 
-7. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>You can try out any command from the [Command Summary](#command-summary) below.
 
-8. Refer to the [Features](#features) below for details of each command.
+The application should be filled with sample data. Additionally, the application should have created a `/data/` folder and some additional files in the same folder as `peoplesoft.jar`. Use the `clear` command to delete the sample data.
+
+### Exploring the sample data
+
+The sample data is meant to help users get started using PeopleSoft. This is a tutorial of some of the basic features of PeopleSoft using the sample data. As such, it is not a comprehensive overview of every feature in the application. You can refer to the [features](#features) section for more information about specific features.<br>
+
+To start off, notice that the sample data contains some employees under the table of employees. Here you can see the details of the employees, including their name, base pay and tags. Notice also that the sample data does not contain any jobs.
+
+#### Create a job
+
+To create a job, you can use the command `add`. You will have to specify a name and a duration using the prefixes `n/` and `d/` respectively. In this case, a aircon repair job that lasts two hours will be created.
+
+1. Type `add n/Repair aircon d/2` in the command window.
+
+This is the command to add a new job with the name 'Repair aircon' and a duration of two hours.
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: The order of the prefixes `n/` and `d/` does not matter.
+
+</div>
+
+2. Hit **enter** to run the command.
+
+A new job should be created in the table of jobs. It should have the name 'Repair aircon' with a duration of '2h'.
+
+#### Assign person to a job
+
+Like in the real-world, jobs need to be assigned to employees. This can be done in PeopleSoft with the command `assign`. `assign` is a command that is **index-based**, this means that it uses the ordering of the items displayed in the table. You will have to specify at least two indexes, one for the job, and at least one for the employees.
```

Comment:
```suggestion
Like in the real world, jobs need to be assigned to employees. This can be done in PeopleSoft with the command `assign`. `assign` is a command that is **index-based**, this means that it uses the ordering of the items displayed in the table. You will have to specify at least two indexes, one for the job, and at least one for the employees.
```